### ISSUE TITLE
Ability to export site with relative links

### ISSUE THREAD

**Issue by derintolu** [WP2Static/static-html-output#137]
why was this feature removed?

**Comment by ayl** [comment]
I agree it would be helpful to export all the resources and posts as relative links. It would help with some of the http/https mixed content invalidation.

**Comment by leonstafford** [comment]
Hi all, there's another issue(s) where I explain some of the reasons (potential for bad exports/invalid HTML). 

I'd still like to offer the ability, whether it requires some disclaimer about the risks.

To avoid mixed protocols, a "force https" plugin may do a better job.

Adding your own post processing step to do additional rewrites is another option and a bit easier in WPStatic's more developer-friendly hooks. WP2Static also lends itself better for me or anyone to create an addon specifically for this user need.

For me, the most appealing benefit to the relative URLs is staging/prod envs using the same generated static site without extra processing, but usually you'll know the exact URL to target for deployment. 

Finishing up some exciting work on https://github.com/leonstafford/lokl, which will aid in testing/developing features like this.

In the meantime, if you need help with some shell scripts to do extra rewrites on your output site's files, share me what you've got so far and can assist.

Thanks for your patience/understanding and I am listening to this request.
